android/res/values/styles.xml

```diff
@@ -193,10 +193,10 @@
     </style>
 
     <!-- Menu Media Type end -->
-
+    <!--v21-->
```

Comment:
This is just a documentation for me so we know that this particular style is used separately for lollipop and later in styles.xml(v21). So that when you update the style you update it in the other version as well.